airbyte-config/models/src/main/resources/types/FailureReason.yaml

```diff
@@ -17,6 +17,7 @@ properties:
       - persistence
       - normalization
       - dbt
+      - connection_workflow
```

Comment:
I added a new FailureOrigin enum value here, since none of the existing ones felt like they fit this use case. Open to feedback on the value name here

Reply:
Can we rename it to internal_error. I think that `connection_wprkflow` can lead to misunderstanding (like make you believe that it is a configuration issue).

Reply:
What do you think about `platform`? That feels like it is more consistent with the other values here as it is sort of a "location" of where the error occurred, rather than a description of the error itself, since these are failure "origin" values

Reply:
can we name it `airbyte_platform`? To make sure that the client understand that it is not cause by his configuration.

Reply:
Sure, makes sense